state/factory/workingset.go

```diff
@@ -122,6 +125,9 @@ func (ws *workingSet) runActions(
 }
 
 func withActionCtx(ctx context.Context, selp action.SealedEnvelope) (context.Context, error) {
+	if !protocol.MustGetFeatureCtx(ctx).EnableWeb3Staking && isWeb3StakingAction(selp) {
```

Comment:
delete this section and add error handling in https://github.com/iotexproject/iotex-core/pull/3345/files#diff-5168c619aa2e8281d3f34e0423560a0a1fda52f9fd1a49e57ba8390f0b3bfbd3R465